### PR TITLE
Revise Abstract and Introduction

### PR DESCRIPTION
**Pull request by bifurcation**

### CODE REVIEW

File: draft-omara-sframe.md [hunk 1]
```diff
@@ -44,26 +44,29 @@ informative:
...
   2. End-to-end (E2E) encryption of media between the endpoints
 
-While DTLS-SRTP can be used as an efficient HBH mechanism, it is inherently point-to-point and therefore not suitable for a SFU context. In addition, given the various scenarios in which video calling occurs, minimizing the bandwidth overhead of end-to-end encryption is also an important goal.
+The Secure Real-Time Protocol (SRTP) is already widely used for HBH encryption {{?RFC3711}}.  The SRTP "double encryption" scheme defines a way to do E2E encryption in SRTP {{?RFC8723}}. Unfortunately, this scheme has poor efficiency and high complexity, and its entanglement with RTP makes it unworkable in several realistic SFU scenarios.  
 
-This document proposes a new end-to-end encryption mechanism known as SFrame, specifically designed to work in group conference calls with SFUs.
+This document proposes a new end-to-end encryption mechanism known as SFrame, specifically designed to work in group conference calls with SFUs.  SFrame is a general encryption framing that can be used to protect payloads sent over SRTP
 
 ~~~~~
```
Comment (martinthomson) [inline draft-omara-sframe.md:68]: Here's a scary experiment:

```suggestion
~~~ aasvg
```

Comment (bifurcation) [reply draft-omara-sframe.md:68]: What is this supposed to do?  Auto-generate SVG from ASCII art?

Comment (martinthomson) [reply draft-omara-sframe.md:68]: Yep.  For example:

image

**Comment by youennf** [comment]
Needs rebasing. I am not sure what the changes should look like, @bifurcation , can you do the rebasing?

File: draft-ietf-sframe-enc.md [hunk 2]
```diff
@@ -54,47 +54,52 @@ informative:
 
 --- abstract
 
-This document describes the Secure Frame (SFrame) end-to-end encryption and authentication mechanism for media frames in a multiparty conference call, in which central media servers (SFUs) can access the media metadata needed to make forwarding decisions without having access to the actual media.
-The proposed mechanism differs from other approaches through its use of media frames as the encryptable unit, instead of individual RTP packets, which makes it more bandwidth efficient and also allows use with non-RTP transports.
+This document describes the Secure Frame (SFrame) end-to-end encryption and authentication mechanism for media frames in a multiparty conference call, in which central media servers (selective forwarding units or SFUs) can access the media metadata needed to make forwarding decisions without having access to the actual media.
+
+The proposed mechanism differs from the Secure Real-Time Protocol (SRTP) in that
+it is independent of RTP (thus compatible with non-RTP media transport) and can
+be applied to whole media frames in order to be more bandwidth efficient.
```
Comment (youennf) [inline draft-ietf-sframe-enc.md:61]: This one is 80 char wrapped, not the other section.
We should probably try to be consistent.

File: draft-ietf-sframe-enc.md [hunk 3]
```diff
@@ -54,47 +54,52 @@ informative:
...
 
 --- middle
 
 
 # Introduction
-Modern multi-party video call systems use Selective Forwarding Unit (SFU) servers to efficiently route RTP streams to call endpoints based on factors such as available bandwidth, desired video size, codec support, and other factors. In order for the SFU to work properly though, it needs to be able to access RTP metadata and RTCP feedback messages, which is not possible if all RTP/RTCP traffic is end-to-end encrypted.
```
Comment (youennf) [inline draft-ietf-sframe-enc.md:64]: Do we need to spell out SFU since we already clarified what this is in the abstract?

**Review by youennf — APPROVED** [review]

File: draft-ietf-sframe-enc.md (hunk 3 above)
Comment (bifurcation) [reply draft-ietf-sframe-enc.md:64]: Seems like it doesn't hurt, and might improve clarity if someone skips the abstract.

File: draft-ietf-sframe-enc.md (hunk 2 above)
Comment (bifurcation) [reply draft-ietf-sframe-enc.md:65]: Updated the sections affected by this PR.  I can do a global 80-char-wrap PR once the current queue drains.

**Pull request merged**